WorkloadManifest.json is missing AOT entries for Mac Catalyst

### Description

This happens when building for Mac Catalyst with .NET 6 Preview 7: 

> /Users/rolf/work/maccore/mono-master/xamarin-macios/builds/downloads/dotnet-sdk-6.0.100-preview.7.21321.2-osx-x64/sdk-manifests/6.0.100/microsoft.net.workload.mono.toolchain/WorkloadManifest.targets(29,37): error MSB4236: The SDK 'Microsoft.NETCore.App.Runtime.AOT.Cross.maccatalyst-x64' specified could not be found.

I believe it should be listed here: https://github.com/dotnet/runtime/blob/63c8ab2e7acedca0547bc9d59ba8b7d34faf3ddb/src/mono/nuget/Microsoft.NET.Workload.Mono.Toolchain.Manifest/WorkloadManifest.json.in

This is blocking our bump to P7 (which contains a good chunk of P6 too, since we've been blocked for other reasons for a while already).

binlog: 
[msbuild-log.binlog.zip](https://github.com/dotnet/runtime/files/6687301/msbuild-log.binlog.zip)

CC @steveisok

--- a/src/mono/nuget/Microsoft.NET.Workload.Mono.Toolchain.Manifest/WorkloadManifest.targets
+++ b/src/mono/nuget/Microsoft.NET.Workload.Mono.Toolchain.Manifest/WorkloadManifest.targets
@@ -26,8 +26,7 @@
 
     <ImportGroup Condition="'$(TargetPlatformIdentifier)' == 'maccatalyst'">
         <Import Project="Sdk.props" Sdk="Microsoft.NET.Runtime.RuntimeConfigParser.Task" />
-        <Import Project="Sdk.props" Sdk="Microsoft.NETCore.App.Runtime.AOT.Cross.maccatalyst-x64" />
-        <Import Project="Sdk.props" Sdk="Microsoft.NETCore.App.Runtime.AOT.Cross.maccatalyst-arm64" />
+        <Import Project="Sdk.props" Sdk="Microsoft.NETCore.App.Runtime.AOT.Cross.maccatalyst" />
     </ImportGroup>
 
     <ImportGroup Condition="'$(TargetPlatformIdentifier)' == 'tvos'">

